internal/pkg/cli/svc_deploy.go

```diff
@@ -118,13 +118,14 @@ func newSvcDeployer(o *deploySvcOpts) (workloadDeployer, error) {
 	content := o.appliedManifest.Manifest()
 	var deployer workloadDeployer
 	in := clideploy.WorkloadDeployerInput{
-		SessionProvider: o.sessProvider,
-		Name:            o.name,
-		App:             targetApp,
-		Env:             o.targetEnv,
-		ImageTag:        o.imageTag,
-		Mft:             content,
-		RawMft:          raw,
+		SessionProvider:   o.sessProvider,
+		Name:              o.name,
+		App:               targetApp,
+		Env:               o.targetEnv,
+		ImageTag:          o.imageTag,
+		Mft:               content,
+		RawMft:            raw,
+		UploadAddonAssets: false, // TODO(dnrnd): change to true to enable packaging addons
```

Comment:
qq before I review the rest: will this be a flag in `deploy` and `package` commands or will this always be true?

Reply:
this feature will not add any new flags. on `svc deploy`, it will always upload the local assets. for `svc package` it will respect the `--upload-assets` flag.